isoutils/boot.txt

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Given the relatively small size, I would prefer we just write this file with a native go API instead of storing and copying this file.

Reply:
Done.